Client/Ecosia/UI/EcosiaHome.swift

```diff
@@ -59,6 +59,11 @@ final class EcosiaHome: UICollectionViewController, UICollectionViewDelegateFlow
                     return "treeCounter"
                 }
             }
+
+            var label: Analytics.Label.Navigation? {
+                if self == .treeCount { return .counter }
```

Comment:
I see you updated the switch, but you still added a conditional. Maybe I'm missing something, but is there a case when self != .treeCount?

Reply:
good point.. I streamlined the other getters also